### DISCUSSION TITLE
List of apps made with Tauri

### DISCUSSION THREAD

**Discussion by trymeouteh** [tauri-apps/tauri#3969] [Ideas]
On the Electron website, they have a section which shows what apps are powered by Electron. Please add a similar section to the Tauri website.

**Comment by azerpas** [comment]
There's currently a Github repository for the matter:
https://github.com/tauri-apps/awesome-tauri

**Reply by FabianLars** [reply]
We used to have a showcase page on our website but [removed it a while back](https://github.com/tauri-apps/tauri-docs/pull/399) in favor of the awesome-tauri. We are generally not opposed to re-introducing it at some point, but it won't happen in the foreseeable future.
If you're on our discord server you can read through the convo that lead to removing it ~[here](https://discord.com/channels/616186924390023171/662624589037436928/939167092953972836).
